packages/js/src/generators/library/schema.json

```diff
@@ -14,13 +14,19 @@
         "index": 0
       },
       "x-prompt": "What name would you like to use for the library?",
-      "pattern": "^[a-zA-Z].*$"
+      "pattern": "(?:^@[a-zA-Z0-9-*~][a-zA-Z0-9-*._~]*\\/[a-zA-Z0-9-~][a-zA-Z0-9-._~]*|^[a-zA-Z][^:]*)$"
     },
     "directory": {
       "type": "string",
       "description": "A directory where the lib is placed.",
       "x-priority": "important"
     },
+    "nameDirectoryFormat": {
+      "description": "Whether to generate the name and directory as provided (`as-provided`) or generate them composing their values and taking the configured layout into account (`derived`).",
+      "type": "string",
+      "enum": ["as-provided", "derived"],
+      "x-priority": "important"
```

Comment:
I think this should actually be hidden.

Reply:
I agree. With the discussed changes for Nx Console, it's no longer important.